rs/tests/consensus/orchestrator/cup_compatibility_test.rs

```diff
@@ -125,9 +125,7 @@ fn download_mainnet_binary(version: String, log: &Logger, target_dir: &Path) ->
 fn test(env: TestEnv) {
     let log = env.logger();
 
-    let mainnet_version =
-        read_dependency_to_string("mainnet_nns_subnet_revision.txt").expect("mainnet IC version");
-    info!(log, "Continuing with mainnet version {mainnet_version}");
+    let mainnet_version = get_mainnet_nns_revision();
```

Comment:
I think this log was deleted accidentally
```suggestion
    let mainnet_version = get_mainnet_nns_revision();
    info!(log, "Continuing with mainnet version {mainnet_version}");
```

Reply:
My mistake! Fixed!